flutter/test/web/web_sentry_js_binding_test.dart

```diff
@@ -0,0 +1,53 @@
+@TestOn('browser')
+library;
+
+import 'dart:js_interop';
+import 'dart:js_interop_unsafe';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sentry_flutter/src/web/script_loader/sentry_script_loader.dart';
+import 'package:sentry_flutter/src/web/sentry_js_bundle.dart';
+import 'package:sentry_flutter/src/web/web_sentry_js_binding.dart';
+
+import '../mocks.dart';
+
+/// Test file for testing specific binding implementation such as caching.
+/// Most of the other tests that involve the binding live in sentry_web_test.dart.
+void main() {
+  group(WebSentryJsBinding, () {
+    late Fixture fixture;
+
+    setUp(() {
+      fixture = Fixture();
+    });
+
+    test(
```

Comment:
Are there more tests that we need to add here? For example all the cases where we return early.

Reply:
I added some more tests to `sentry_web_test`